doc/admin.md

```diff
@@ -0,0 +1,193 @@
+
+# Installation and Administration Guide
+
+## Introduction
+
+This guide defines the procedure to install the FIWARE Testbed Deploy.
+
+For general information, please refer to GitHub's [README](https://github.com/telefonicaid/fiware-testbed-deploy/blob/develop/README.md).
+
+## Build and install
+
+### Requirements
+- This scripts has been tested on a Debian 7 system, but any other recent Linux
+  distribution with the software described should work
+
+The following software must be installed (e.g. using apt-get on Debian and Ubuntu,
+or with yum in CentOS):
+
+- Python 2.7
+- pip
+- virtualenv
+
+### Installation
+
+The recommend installation method is using a virtualenv. Actually, the installation
+process is only about the python dependencies, because the scripts do not need
+installation.
+
+- 1.- Create a virtualenv 'deleteENV' invoking *virtualenv deleteENV*
+- 2.- Activate the virtualenv with *source deleteENV/bin/activate*
+- 3.- Install the requirements running *pip install -r requirements.txt
+   --allow-all-external*
+
+## Running FIWARE testbed deploy
+
+### Deploying one testbed
+To deploy just a testbed, it is required just execute one script. However some variables should be exported before.
+
+    export OS_AUTH_URL=<the authentication URL for the keystone in the Cloud>
+    export OS_USERNAME=<a user with an account in the Cloud>
+    export OS_TENANT_NAME=<a project name from the account in the Cloud>
+    export OS_PASSWORD=<the password for an account in the Cloud>
+    export OS_REGION_NAME=<the region name>
+    export OS_USER_DOMAIN_NAME=<OpenStack user domain name>
+    export OS_PROJECT_DOMAIN_NAME=<OpenStack project domain name>
+    export BOOKED_IP=<a booked IP in your Cloud infrastructure to deploy the VM>
+    export Region1=<the name for the region of the Cloud to be deployed>
+
+    fiwaretestbeddeploy/launch_vm.py
+
+The *launch_vm.py* ends in a few seconds, showing the floating IP. Then, it is
+ possible to connect to the ubuntu account of the server (using the
+SSH key at *~/.ssh/testbedskuld_key*).
+
+    ssh ubuntu@floating_ip
```

Comment:
You told that the script was tested on Debian7 but here you are using ubuntu user name.